js/Store.js

```diff
@@ -31,6 +31,20 @@ function reducer(state, action) {
     }
     return newState;
 
+  case 'setView':
+    newState.currentView = action.view;
+    return newState;
+
+  case 'seeking':
+    newState.seeking = true;
+    newState.currentView = 'play';
+    return newState;
+
+  case 'register':
```

Comment:
see how the other actions have a comment of the same format saying what fields the action contains? can you document the new actions in a similar manner?